### PR TITLE
feat: DSW-1739 Added icons page to nuxt

### PR DESCRIPTION
**Pull request by JoshuaNg2332**
This PR adds the following functionality:

- Adds the pie icons to the Nuxt 3 app.
- Updates to icons dep across all applications.

Screenshot 2024-02-16 at 16 41 48

### CODE REVIEW

File: nuxt-app/pages/index.vue [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <template>
       <h1>Nuxt - PIE Aperture</h1>
       <NuxtLink to="/form">Form page</NuxtLink>
```
Comment (jamieomaguire) [inline nuxt-app/pages/index.vue:3]: I wonder if these should be pie links

Comment (siggerzz) [reply nuxt-app/pages/index.vue:3]: My PR addresses this :)

File: nuxt-app/pages/icons.vue [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<template>
+    <div>
+      <h1>PIE Icons Test Page</h1>
+      <div class="icon-container">
+        <component v-for="icon in icons" :key="icon" :is="icon" />
+      </div>
+    </div>
+</template>
+  
+<script>
```
Comment (jamieomaguire) [inline nuxt-app/pages/icons.vue:10]: please can you add `setup` as an attribute to the script and add `lang='ts'`?

**Review by xander-marjoram — DISMISSED** [review]

**Comment by JoshuaNg2332** [comment]
Just a general comment, we've not added the visual tests in for this due to the ongoing work with nuxt SSR deployment issues and AWS. Tests will be enabled at a later date.

**Review by xander-marjoram — APPROVED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Pull request merged**